src/fty_discovery_server.cc

```diff
@@ -644,6 +644,13 @@ void ftydiscovery_create_asset(fty_discovery_server_t* self, zmsg_t** msg_p)
             }
         }
 
+        // In device centric view for sts, add missing default value for ext attributes
+        if (self->device_centric && streq(name, "sts")) {
+            fty_proto_ext_insert(asset, "outlet.count", "1");
```

Comment:
Could we add this value in the payload always ?

Reply:
for homogeneity, we should perhaps define here 'outlet.1.label' ext. attr. too.
see my latest fix related to device outlets extraction from ext. attr. in fty-asset-rest (https://github.com/42ity/fty-asset-rest/pull/54)

Reply:
@perrettecl yes It is an error should be added in all case for automatic discovery. For manual discovery, should be checked in fty-discovey-ng.
@jeffdegott I don't know if it is the good place for that. To be discussed i you want.